src/Swashbuckle.AspNetCore.Swagger/ISwaggerDocumentInformation.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.OpenApi.Models;
+
+namespace Swashbuckle.AspNetCore.Swagger
+{
+    public interface ISwaggerDocumentInformation
+    {
+        string[] GetDocumentNames();
```

Comment:
Let's return `IList<string>` so we're abstracting the specific type.

Reply:
Fixed